homeassistant/helpers/entity_platform.py

```diff
@@ -275,17 +275,24 @@ def add_entities(self, new_entities, update_before_add=False):
             device_info = entity.device_info
 
             if config_entry_id is not None and device_info is not None:
+                processed_dev_info = {
+                    'config_entry_id': config_entry_id
+                }
+                for key in (
+                        'connections',
+                        'identifiers',
+                        'manufacturer',
+                        'model',
+                        'name',
+                        'sw_version',
+                        'via_hub',
+                ):
+                    if key in device_info:
```

Comment:
Shouldn't this be `if device_info.get(key) is not None:`?

I think I've seen some device info implementations setting the values of keys to `None` to indicate absence.

Reply:
That should be changed in the implementation. If you set something to None, you indicate that it shouldn't be any value.